src/mutation.js

```diff
@@ -61,12 +61,14 @@ var methods = (Mutation.methods = {
  * @param {string} bytes - Base64 encoded string.
  * @returns {string|number|buffer}
  */
-Mutation.convertFromBytes = function(bytes, options) {
-  var buf = Buffer.from(bytes, 'base64');
-  var num = new Int64(buf).toNumber();
+Mutation.convertFromBytes = function(bytes, options, isPossibleNumber) {
```

Comment:
Can `isPossibleNumber` be `options.isPossibleNumber`?

Reply:
In general, this API should be changed to accept a configuration object so that we don't have the multiple arguments that aren't clear from where the method is called:

```js
Mutation.convertFromBytes(chunk.value, this.options, true)
```

That isn't idiomatic JS, so if modifying `this.options` is off the table, we can instead wrap the options:

```js
Mutation.convertFromBytes(chunk.value, {
  userOptions: this.options, // is that accurate to what these are?
  isPossibleNumber: true
}, true)
```

Additionally, maybe the option name itself could be `checkIfNumber`, to indicate an additional action the method should take if enabled, similar to how `decode` is an action.

Reply:
I've gone the `userOptions` route.